src/commands/convert/burpsuite2hdf.ts

```diff
@@ -4,16 +4,17 @@ import {BurpSuiteMapper as Mapper} from '@mitre/hdf-converters'
 import {checkInput, checkSuffix} from '../../utils/global'
 
 export default class Burpsuite2HDF extends Command {
-  static usage = 'convert burpsuite2hdf -i, --input=XML -o, --output=OUTPUT'
+  static usage = 'convert burpsuite2hdf -i, --input=XML -o, --output=OUTPUT -w, --with-raw'
```

Comment:
Like @Rlin232 mentioned, don't worry about the usage statements for now.  They're going to be drastically revised in an upcoming PR.